### PR TITLE
Installer cache generation fixed

### PR DESCRIPTION
**Pull request by Shudrum**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | On some servers, the cache cannot be correctly generated. This fix it. |
| Type? | bug fix |
| Category? | IN |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | BOOM-781 |
| How to test? | Manually clear the cache folder, entirely, the reinstall. |

### CODE REVIEW

**Comment by Shine-neko** [comment]
@Shudrum Did you see that? https://github.com/PrestaShop/PrestaShop/pull/5316 ?

**Comment by Shudrum** [comment]
@Shine-neko how … missed it ><

File: install-dev/models/install.php [hunk 1]
```diff
@@ -123,18 +139,31 @@ public function generateSettingsFile($database_host, $database_user, $database_p
...
+
+        $sf2Refresh = new \PrestaShopBundle\Service\Cache\Refresh('dev');
+        $sf2Refresh->addCacheClear();
+        $output = $sf2Refresh->execute();
+
+        if (0 !== $output['cache:clear']['exitCode']) {
+            $this->setError(explode("\n", $output['cache:clear']['output']));
+            return false;
+        }
+
```
Comment (Shudrum) [inline install-dev/models/install.php]: This need a little bit of factorization.

**Comment by xBorderie** [comment]
@Shine-neko Does this mean #5316 can be closed?

**Pull request merged**